Source/WebCore/page/win/ResourceUsageThreadWin.cpp

```diff
@@ -129,6 +129,14 @@ void ResourceUsageThread::platformCollectMemoryData(JSC::VM* vm, ResourceUsageDa
     data.categories[MemoryCategory::GCOwned].dirtySize = currentGCOwnedExtra - currentGCOwnedExternal;
     data.categories[MemoryCategory::GCOwned].externalSize = currentGCOwnedExternal;
 
+    usage -= currentGCHeapCapacity;
+    // Following ResourceUsageThreadCocoa implementation
+    auto currentGCOwnedGenerallyInMalloc = currentGCOwnedExtra - currentGCOwnedExternal;
+    if (currentGCOwnedGenerallyInMalloc < mallocBucket.dirtySize)
```

Comment:
I think you forgot to change `mallocBucket.dirtySize` to `usage` on this line.

Reply:
Sorry, that was missing when creating a new branch.